docs/src/main/sphinx/release/release-403.md

```diff
@@ -0,0 +1,62 @@
+# Release 403 (12 Nov 2022)
+
+## General
+
+* Add the amount of data read from external sources during table scans to
+  `EXPLAIN ANALYZE`. ({issue}`14907`)
+* Fix handling of `time(n) with time zone` when clients do not support variable
+  precision temporal types. ({issue}`14950`)
+* Fix query deadlock in multi-join queries where broadcast join size is
+  underestimated. ({issue}`14948`)
+* Fix incorrect results when `min(x, n)` or `max(x, n)` is used as a window
+  function. ({issue}`14886`)
+* Fix failure for certain queries involving joins over partitioned tables. ({issue}`14317`)
+* Fix incorrect order of parameters in `DESCRIBE INPUT` when they appear in a
+  `WITH` clause. ({issue}`14738`)
+
+## Security
+
+* Allow access token passthrough when using OAuth 2.0 with refresh tokens
+  enabled. ({issue}`14949`)
+
+## Fault-tolerant execution
```

Comment:
Move this to the general section. We don’t usually create dedicated sections for each feature (aside from higher level components like connectors, clients, web ui, or general areas like the SPI or security)